packages/de1/tests/de1/events.test.ts

```diff
@@ -0,0 +1,61 @@
+import { EventEmitter } from "events";
+import { Service } from "@sblendid/sblendid";
+import DE1 from "../../src/de1";
+import converters, { Name } from "../../src/converters";
+import events from "../__fixtures__/events";
+
+describe("de1 events", () => {
+  const de1 = new DE1();
+  const emitter = new EventEmitter();
+
+  // todo we don't have enough events
+  // todo this mapping code is horrible
+  const namesWithEventFixture = Object.keys(events)
+    .map(uuid => Object.entries(converters).find(([, c]) => c.uuid === uuid))
+    .map(c => c![0] as Name);
```

Comment:
Funky! :D

Reply:
I‘m sorry 🤦🏻‍♂️

Reply:
Just tired and wanna be done already 🤦🏻‍♂️